packages/web-react/src/types/pill.ts

```diff
@@ -1,7 +1,7 @@
 import { ElementType, JSXElementConstructor } from 'react';
-import { ChildrenProps, TransferProps } from './shared';
+import { ActionColor, ChildrenProps, EmotionColor, TransferProps } from './shared';
 
-export type PillColor = 'selected' | 'danger' | 'informative' | 'success' | 'warning';
+export type PillColor = 'selected' | 'unselected';
```

Comment:
```suggestion
export type PillColor<C> = ActionColor<void> | EmotionColor<void> | 'selected' | 'unselected' | C;
```

Is `selected` and `unselected` a color? Isn't it that another dictionary?

Reply:
They are colors, and they are not a dictionary. They are an exception.